tests/tests_basic.py

```diff
@@ -12,3 +12,149 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import pytest
+import subprocess
+
+import os.path
+import hashlib
+
+###################
+# Setup path data #
+###################
+
+tests_path = os.path.dirname( os.path.abspath( __file__ ))
+samples_path = os.path.join( tests_path, 'samples/')
+base_path = os.path.abspath( os.path.join( tests_path, '../' ))
+scripts_path = os.path.join( base_path, 'scripts/')
+
+####################
```

Comment:
That would require a change in the way the imports/specific files the tests run on work; depending on how we eventually set it up it's definitely something to consider though. Right now, I'm fairly happy with them being in the same file up until we get larger sets of tests. At that point, it's likely that a lot of the setup and helpers would go into conftest.py, which is used by pytest.